react/components/ProductSummaryAddToListButton/ProductSummaryAddToListButton.js

```diff
@@ -2,11 +2,12 @@ import React, { useMemo } from 'react'
 import { ExtensionPoint } from 'vtex.render-runtime'
 import { path } from 'ramda'
 import { useProductSummary } from 'vtex.product-summary-context/ProductSummaryContext'
-import productSummary from '../../productSummary.css'
+import { useCssHandles } from 'vtex.css-handles'
 
 // TODO: change ProductSummaryContext to have `selectedSku` field instead of `sku`
```

Comment:
Is this an issue in the repo?

Reply:
Not really, but since this information is in two places, it would be nice if we got this from only one on every app.